src/Compilers/Core/Portable/RuleSet/RuleSetInclude.cs

```diff
@@ -57,6 +57,12 @@ public RuleSet LoadRuleSet(RuleSet parent)
                 path = GetIncludePath(parent);
                 ruleSet = RuleSetProcessor.LoadFromFile(path);
             }
+            catch (FileNotFoundException)
+            {
+                // The compiler uses the same rule set files as FxCop, but doesn't have all of
```

Comment:
@tmeschter Instead of completely ignoring the exception, we may want to consider reporting the InvalidRuleset diagnostic with severity "Info", so that the underlying issue is reported in error list (and also in output window for verbose builds).

This is what we do for analyzer exception diagnostics. We don't want them to break the build, even with warnaserror set to true, but do want the information to surface to the user in some way.